pkgs/servers/tailscale/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute }:
+
+buildGoModule rec {
+  pname = "tailscale";
+  version = "0.96-33";
+
+  src = fetchFromGitHub {
+    owner = "tailscale";
+    repo = "tailscale";
+    rev = "19cc4f8b8ecfdc16136d8489a1c2b899f556fda7";
+    sha256 = "0kcf3mz7fs15dm1dnkvrmdkm3agrl1zlg9ngb7cwfmvkkw1rkl6i";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  CGO_ENABLED = 0;
+
+  goPackagePath = "tailscale.com";
+  modSha256 = "1pjqfzw411k6kw8hqf56irnlhnl8947p1ad8yd84zvqqpzfs3jmz";
+  subPackages = [ "cmd/tailscale" "cmd/tailscaled" ];
+
+  postInstall = ''
+    wrapProgram $out/bin/tailscaled --prefix PATH : ${
+      lib.makeBinPath [ iproute iptables ]
+    }
+  '';
+
+  meta = with lib; {
+    homepage = "https://tailscale.com";
+    description = "The easiest, most secure way to use WireGuard and 2FA";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ danderson mbaillie ];
```

Comment:
Please add platforms.

Reply:
@martinbaillie I believe `platforms.linux` does what we want for now. Tailscale should work on all the CPU architectures that Nix on linux supports. We could add darwin in the next couple weeks, but I'm not sure it makes sense outside of NixOS.

Reply:
Agreed, cheers. Have added `platforms.linux`.